.github/workflows/test-base.yml

```
@@ -4,71 +4,117 @@ on:
       goodmap-version:
         required: true
         type: string
+        default: main
```

actionlint (1.7.8)

input "goodmap-version" of workflow_call event has the default value "main", but it is also required. if an input is marked as required, its default value will never be used

(events)